great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -267,6 +290,49 @@ def config_variables(self) -> Dict:
             self._config_variables = self._load_config_variables()
         return self._config_variables
 
+    @property
+    def config(self) -> DataContextConfig:
+        return self._project_config
+
+    @property
+    def root_directory(self) -> Optional[str]:
+        """The root directory for configuration objects in the data context; the location in which
+        ``great_expectations.yml`` is located.
+
+        Why does this exist in AbstractDataContext? CloudDataContext and FileDataContext both use it
+
+        """
+        if hasattr(self, "_context_root_directory"):
+            return self._context_root_directory
+        return
```

Comment:
why `hasattr` here? and nitpick but I don't think we need the return on 307

Reply:
I'd like to see this resolved before merging the PR

Reply:
I'd like to see this resolved before merging the PR